packages/@lwc/engine-core/src/framework/hydration.ts

```diff
@@ -93,16 +85,17 @@ function hydrateNode(node: Node, vnode: VNode): Node | null {
             hydratedNode = hydrateCustomElement(node, vnode);
             break;
     }
-
+    const { nextSibling } = getRendererFromVM(vnode.owner);
```

Comment:
Caridy, can you add a comment here explaining why we are getting renderer from the vnode owner here.
Actually for this method, I feel like we should pass the `owner` as an argument and add the following comment
```
// Hydrating the owner's template, use the owner's renderer
```
In the future anybody contributing to this routine will wonder when to get renderer off of a vnode v/s an vnode.owner.